packages/components/src/search/autocompleters/customers.js

```diff
@@ -78,7 +76,7 @@ export default {
 	getOptionCompletion( customer ) {
 		return {
 			id: customer.id,
-			label: getName( customer ),
+			label: customer.name,
```

Comment:
Not sure about UI implications here, but what do you think about adding the email next to customer name?

Related to above comment about customer's without associated names, but would also help us distinguish between same names (when customer creates multiple accounts under different emails or checkout last name field has been removed).

<img width="472" alt="screen shot 2019-02-13 at 2 59 39 pm" src="https://user-images.githubusercontent.com/10561050/52692843-0106b380-2fa0-11e9-951f-fef818b711d3.png">

Reply:
I'm good with that so long as design is.. what say you @josemarques @timmyc ?

Reply:
Building on the comment above, I think we leave it as-is for now, and get more feedback from Ze and David when he is back next week before we make any further additions/changes right there.